app/controllers/spree/credit_cards_controller.rb

```diff
@@ -63,13 +63,6 @@ def destroy_at_stripe
       stripe_customer&.delete unless stripe_customer.deleted?
     end
 
-    def stripe_account_id
```

Comment:
Did you remove the use of this method within this PR or was it dead code already?

Reply:
Dead code already, or at least I think it was; it was a private method and referenced nowhere else in the file.